encfs/main.cpp

```diff
@@ -718,15 +718,19 @@ static void *idleMonitor(void *_arg) {
   pthread_mutex_lock(&ctx->wakeupMutex);
 
   while (ctx->running) {
-    int usage = ctx->getAndResetUsageCounter();
+    int usage, openCount;
+    ctx->getAndResetUsageCounter(&usage, &openCount);
 
     if (usage == 0 && ctx->isMounted())
       ++idleCycles;
-    else
+    else {
+      if (idleCycles >= timeoutCycles)
+        RLOG(INFO) << "Filesystem no longer inactive: "
+                   << arg->opts->mountPoint;
```

Comment:
Let's imagine the FS was inactive `(idleCycles >= timeoutCycles)`, but unmount failed for some reason.
FS is then still mounted.
Activity to the FS comes back, we then log a "Filesystem no longer inactive" message (so only if it has been inactive before :)